__tests__/app.test.js

```diff
@@ -441,5 +441,44 @@ describe('/api/articles - topic query', () => {
         .then(({ body }) => {
             expect(body.msg).toBe('not found');
         });
-    });             
+    });            
 });
+
+describe('/api/articles/:article_id - comment_count', () => {
+    test('200 respond with an article with the count of all the comments with the article_id', () => {
+        return request(app)
+        .get('/api/articles/1?comment_count')
```

Comment:
This first test is probably the only one we need here, but it should still pass when we make the following request: .get('/api/articles/1') there is no need to add ?comment_count to the path